orca-redis/src/main/java/com/netflix/spinnaker/orca/pipeline/persistence/jedis/RedisExecutionRepository.java

```diff
@@ -464,8 +464,8 @@ public void delete(@Nonnull ExecutionType type, @Nonnull String id) {
   }
 
   @Override
-  public @Nonnull
-  Observable<Execution> retrieveOrchestrationsForApplication(@Nonnull String application, @Nonnull ExecutionCriteria criteria) {
+  public @Nonnull Observable<Execution> retrieveOrchestrationsForApplication(@Nonnull String application,
+                                                                             @Nonnull ExecutionCriteria criteria) {
```

Comment:
Is this your preferred way to do multi line function declarations with long names?

Reply:
This is consistent with most of the Java/Groovy code in our projects. Kotlin has a different standard, which I'm fine with (new lining the first argument and indenting one). The key thing, for me, is not going over 120 characters where feasible.

I really dislike side-scrolling. :)